android/build.gradle

```diff
@@ -144,10 +145,13 @@ repositories {
     mavenCentral()
 }
 
-def kotlin_version = getExtOrDefault('kotlinVersion', project.properties['RNSAC_kotlinVersion'])
+def kotlin_version = safeExtGet('kotlinVersion')
 
 dependencies {
-    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    // For < 0.71, this will be from the local maven repo
+    // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
+    //noinspection GradleDynamicVersion
+    implementation "com.facebook.react:react-native:+"
```

Comment:
are you sure of this ?
Here it takes last react native version.
There was an issue few month ago, the fix was to use a fixed version (like before this patch)

Reply:
Yes, I take it from react native library template ('npx create-react-native-library@latest')